src/components/ErrorComponent.tsx

```diff
@@ -0,0 +1,10 @@
+export function ErrorComponent() {
+  return (
+    <div className="pt-14 flex justify-center items-center flex-col h-full text-neutral-200">
+      <h2 className="text-6xl font-semibold">404</h2>
```

Comment:
The 404 heading doesn't show on wider screens

Reply:
I am working on this with my next branch for responsive design